### ISSUE TITLE
Add a subcommand converting Snowflake Loader manifest into Processing manifest

### ISSUE THREAD

**Issue by chuwy** [snowplow-incubator/snowplowctl#1]

**Comment by alexanderdean** [comment]
Is this a one-time migration? If so, I would put this under `snowplowctl migrations snowflake` or similar...

**Comment by chuwy** [comment]
Yes, this is one-time migration. I'd like to have this ability to perform one-time transformations at least in first couple of versions of processing manifest, while record format is not stabilized and processing manifest used only in batch pipeline (table is not too big).
